test/unit/global-search.v2.test.js

```diff
@@ -17,7 +17,7 @@
 // need to import the whole package to mock getAuthenticatorFromEnvironment
 const sdkCorePackage = require('ibm-cloud-sdk-core');
 
-const { NoAuthAuthenticator } = sdkCorePackage;
```

Comment:
Please install the latest version of the SDK generator and re-gen your code.  There should be no changes to this unit test file.

Note: recent changes were made to the node core library and the SDK generator to separate out some unit test utilities into their own npm package (`@ibm-cloud/sdk-test-utilities`).  By installing the latest generator version, you should get the correct code generated in the unit tests.

Reply:
Please disregard my previous comment.  I have updated the two unit tests manually

Reply:
Resolved by @padamstx